app/components/SearchInput/Input.tsx

```diff
@@ -33,6 +33,7 @@ export const SearchInput = ({onChange, expandable, placeholderText}: SearchInput
         <MagnifyingIcon className="iconsMagnifyingGlass" />
         <input
           name="searchbar"
+          autoComplete="off"
```

Comment:
why?

Reply:
Sorry, I should have explained. It stops this from happening.
<img width="1512" alt="image" src="https://github.com/StampyAI/stampy-ui/assets/148247390/71fae015-a075-4da3-9361-34e0b5efa597">
I remembered Melissa saying she didn't like the autocomplete suggestions, because it covered up the search results. In hindsight though I don't actually think its that bad, because it does serve a purpose of remembering previous searches. What do you think?

Reply:
better to have just one or the other